### DISCUSSION TITLE
AdGuardHome.yaml - Wrong TimeZone - "log:"

### DISCUSSION THREAD

**Discussion by Freebase394** [AdguardTeam/AdGuardHome#6014] [Q&A] [answered]
Hi dear users,

I tried searching and  find what I wanted in the documentation! 
But this DOCS are insuficient for me: [HERE](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file)
I ask for help from anyone who can help, please! 

Can anyone tell me if there is documentation for the following in "AdGuardHome.yaml" :

log:
  file: ""
  max_backups: 0
  max_size: 100
  max_age: 3
  compress: false
  local_time: false  ---->>>> local_time (since v0.107.34): If the time used for formatting the timestamps in is the computer's local time (default: false [UTC]).   How can i use this??? Lixe this example: "local_time: true +1" ?
  verbose: false
  
  
 NOTE:
But finally, in order to be able to make the Logs print the corresponding local time in the file and logs, we have to configure that in system, in my specific case this time, that is the "Alpine OS Linux" [HERE](https://wiki.alpinelinux.org/wiki/Setting_the_timezone)
But finally, in order to be able to make the Logs print the corresponding local time in file, we have to configure that in system, in my specific case that is the "Alpine OS Linux"
Now I have the correct timestamp to my time zone.

**Comment by ppfeufer** [comment, marked as answer]
I would guess to just set it to `true` instead of `false`.
